extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -372,25 +391,33 @@ class AmpAccordion extends AMP.BaseElement {
    * @private
    */
   animateCollapse_(section) {
-    let height, duration;
-    // Collapse the div portion of the section and not the header
-    const sectionChild = section.children[1];
-    return this.measureElement(() => {
-      height = section./*OK*/offsetHeight;
+    let sectionHeight;
+    let headerHeight;
+    let duration;
+    const sectionHeader = section.children[0];
```

Comment:
Personal preference is to use `.firstElementChild` in this case. That's also favored in some parts of the codebase but I can't argue for either. Up to you.